docs/howto/outputting-pdf.txt

```diff
@@ -78,11 +78,12 @@ mention:
   binary content.
 
 * When ``as_attachment=True`` is passed to ``FileResponse``, it sets the
-  appropriate ``Content-Disposition`` header and that tells Web browsers to
-  pop-up a dialog box prompting/confirming how to handle the document even if a
-  default is set on the machine. If the ``as_attachment`` parameter is omitted,
-  browsers will handle the PDF using whatever program/plugin they've been
-  configured to use for PDFs.
+  appropriate ``Content-Disposition`` header to ``attachment``, and that tells
+  Web browsers to pop-up a dialog box prompting/confirming how to handle the
+  document even if a default is set on the machine. If the ``as_attachment``
+  parameter is ommited or ``False``, ``Content-Disposition`` is set to
```

Comment:
Not sure about the _"ommited or `False`"_. I kind of think just _"Otherwise browsers will handle..."_ would read just as well. 

(Given that, I'm not sure we need this change here...?)

Reply:
Suits me, reverted.